CHANGELOG.md

```diff
@@ -2,6 +2,164 @@
 
 ## [Unreleased](https://github.com/rerun-io/rerun/compare/latest...HEAD)
 
+## [0.13.0](https://github.com/rerun-io/rerun/compare/0.12.1...0.13.0) - Fast time series, improved layout editing & UI overrides - 2024-02-07
+
+### Overview & Highlights
+
+This release focuses on scalar time series -- both from a performance and UI perspectives.  
+Check out our [associated blog post](TODO) for more information.
+
+- 📈 Rerun can now visualize many time series in the kHz range in real-time:
+    - The new [query cache](https://github.com/rerun-io/rerun/pull/4856) optimizes data access, improving query performance by 20-50x
```

Comment:
note sure why there's a PR link here; either link to the new crate or dont link to anything